service-worker/README.md

```
@@ -0,0 +1,34 @@
+Installation
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)